dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java

```diff
@@ -237,8 +237,8 @@ public TaskResponse run(String execCommand) throws IOException, InterruptedExcep
             result.setExitStatusCode(process.exitValue());
 
         } else {
-            logger.error("process has failure , exitStatusCode:{}, processExitValue:{}, ready to kill ...",
-                    result.getExitStatusCode(), process.exitValue());
+            logger.error("process has failure , the task timeout configuration value is:{}, ready to kill ...",
+                    result.getExitStatusCode(), taskRequest.getTaskTimeout());
```

Comment:
What you expected to happen? Only print task timeout?

Reply:
When the process.waitFor execution times out and the process process is still alive, an exception will be thrown when the process.exitValue() method is executed. I don't think it is necessary to print process.exitValue here, because this information is already printed later.


https://github.com/apache/dolphinscheduler/blob/a868259c47632c52f808b28c8fe15407514a299e/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java#L228


https://github.com/apache/dolphinscheduler/blob/a868259c47632c52f808b28c8fe15407514a299e/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java#L246

Reply:
@zhuangchong Do you miss `{}` in your error message string?

Reply:
> @zhuangchong Do you miss `{}` in your error message string?

Thanks, it has been changed.